Update project to work with Docker Composer

gregkare opened the pull request:
However sockethub is broken because NPM dependencies are a disaster

gregkare committed:
Update project to work with Docker Composer

However sockethub is broken because NPM dependencies are a disaster

raucao commented:
I thought the Sockethub branch is outdated and everything is in master now.

@silverbucket @galfert Is that right?

galfert commented:
Yes, IIRC the `experimental_v1_0` branch got merged into master already.

gregkare committed:
Fix Docker compose config

Add a linked container for Redis, linked to sockethub
This needs to be tested with sockethub's fix_dockerfile branch and
hyperchannel's fix_docker

gregkare commented:
If you're using docker-machine (on OS X, for example), you'll need to run this for sockethub to be accessible on localhost:10550:

```
$ docker-machine ssh default -N -L 10550:localhost:10550
```

You get different IPs for the VM apparently, in my case it was 192.168.99.100. How to find it:

```
$ docker-machine ls
```

Then I could load hyperchannel from http://192.168.99.100:4200/

I tried to add config to hyperchannel to set the url of the sockethub server but I didn't go far
